lib/modules/reducer.js

```diff
@@ -14,6 +15,7 @@ export default (state=DEFAULT, action={}) => {
       const relevantHistory = state.history.slice(0, state.currentPageIndex + 1);
 
       return {
+        ...state,
```

Comment:
reddit-mobile breaks when this deep merges - it fails loading postsLists from cache, because the parameters change. I haven't fully worked out _why_, so I did the minimum changes here to get shell working, and thought I'd follow up with a fix for both in a separate pr. I can fix it here if you'd prefer.